app/uk/gov/hmrc/agentstatuschange/connectors/AgentServicesAccountConnector.scala

```diff
@@ -62,18 +62,4 @@ class AgentServicesAccountConnector @Inject()(appConfig: AppConfig,
         throw new NotFoundException(s"agency name not found for arn: $arn")
     }
```

Comment:
Now that we've got a DESConnector in this repo that uses "API 1170 (API 4) - Get Agent Record" we can just use that with either an ARN or a UTR

Reply:
👍